pkg/input/personio/personio.go

```diff
@@ -0,0 +1,53 @@
+package personio
+
+import (
+	"context"
+	"sort"
+
+	personiov1 "github.com/giantswarm/personio-go/v1"
+)
+
+type Employee struct {
+	FirstName    string
+	LastName     string
+	Email        string
+	GithubHandle string
+}
+
+// Returns information on active employees from personio.
+func GetActiveEmployees(ctx context.Context, clientID, clientSecret string) ([]Employee, error) {
+	client, err := personiov1.NewClient(ctx, personiov1.DefaultBaseUrl, personiov1.Credentials{
+		ClientId:     clientID,
+		ClientSecret: clientSecret,
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	employees, err := client.GetEmployees()
+	if err != nil {
+		return nil, err
+	}
+
+	var result []Employee
+	for _, employee := range employees {
+		// only return active employees
+		if *employee.GetStringAttribute("status") != "active" {
+			continue
+		}
+
+		result = append(result, Employee{
+			FirstName:    *employee.GetStringAttribute("first_name"),
+			LastName:     *employee.GetStringAttribute("last_name"),
+			Email:        *employee.GetStringAttribute("email"),
+			GithubHandle: *employee.GetStringAttribute("dynamic_3196204"),
```

Comment:
One could make this a contant defined at the top of the file?

How the GitHub login field name (here: `dynamic_3196204`) differs between Personio instances.

Reply:
Right, done